content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -27,63 +27,136 @@ The installation tool uses the `eksctl` command and doesn't support the `--profi
+Notes:
 
-    ```shell
-    aws iam list-roles \
-        | jq -r ".Roles[] \
-        | select(.RoleName \
-        | startswith(\"eksctl-$AWS_CLUSTER_NAME\") and contains(\"NodeInstanceRole\")) \
-        .RoleName"
+* **${CONFIG_URI}** - The GitHub address of the configuration YAML file that
+  you want to use to deploy Kubeflow. For AWS deployments, the following
+  configurations are available:
 
-    eksctl-kubeflow-example-nodegroup-ng-185-NodeInstanceRole-1DDJJXQBG9EM6
-    ```
+  * `{{% config-uri-aws-standard %}}` 
+  * `{{% config-uri-aws-cognito %}}`
 
-    Change cluster region and worker roles names in your `kfctl_aws.yaml`
-    ```yaml
-      region: us-west-2
-      roles:
-        - eksctl-kubeflow-example-nodegroup-ng-185-NodeInstanceRole-1DDJJXQBG9EM6
-    ```
-    > If you have multiple node groups, you will see corresponding number of node group roles. In that case, please provide the role names as an array.
+    When you run `kfctl apply` or `kfctl build` (see the next step), kfctl creates
+    a local version of the configuration YAML file which you can further
+    customize if necessary.
 
-1. Run the following commands to set up your environment and initialize the cluster.
+* **${KF_NAME}** - The name of your Kubeflow deployment.
+  You should set this value to be the same as your AWS cluster name.
+  The value of KF_NAME must consist of lower case alphanumeric characters or
+  '-', and must start and end with an alphanumeric character.
+  The value of this variable cannot be greater than 25 characters. It must
+  contain just a name, not a directory path.
+  This value also becomes the name of the directory where your Kubeflow 
+  configurations are stored, that is, the Kubeflow application directory. 
+  
+* **${KF_DIR}** - The full path to your Kubeflow application directory.
 
-    ```shell
-    kfctl init ${KFAPP} --config=${CONFIG} -V
-    cd ${KFAPP}
+## Set up your Kubeflow configuration
 
-    kfctl generate all -V
-    kfctl apply all -V
-    ```
+Run kfctl to build your configuration files, so that you can edit the 
+configuration before deploying Kubeflow:
+
+1. Run the `kfctl build` command to set up your configuration:
+
+  ```
+  mkdir -p ${KF_DIR}
+  cd ${KF_DIR}
+  kfctl build -V -f ${CONFIG_URI}
+  ```
+
+1. Set an environment variable pointing to your local configuration file:
 
-    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`. It will be used as eks cluster name.
-    * CONFIG - Path to the configuration file
+  ```
+  export CONFIG_FILE=${KF_DIR}/kfctl_aws.yaml
+  ```
+
+    Or:
+
+  ```
+  export CONFIG_FILE=${KF_DIR}/kfctl_aws_cognito.yaml
+  ```
+
+## Customize your configuration
+
+1. Retrieve the AWS Region and IAM role name for your worker nodes.
```

Comment:
@sarahmaddox Can you add one more line here in this section?

1. Replace AWS cluster name in the config file.
```
sed -i '.bak' -e 's/kubeflow-aws/'"$AWS_CLUSTER_NAME"'/' ${CONFIG_FILE}
```

Reply:
Done.